docs/001_develop/03_client-capabilities/007_forms/002_smart-forms/06_conditional-filtering.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: 'Foundation Forms - Conditional Filtering'
+sidebar_label: 'Conditional Filtering'
+id: foundation-forms-conditional-filtering
+keywords: [genesis, foundation, ui, forms, smart forms, conditional filtering]
+tags:
+  - genesis
+  - foundation
+  - ui
+  - forms
+  - smart forms
+  - conditional filtering
+---
+
+import ConditionalFilteringExample from './examples/conditional_filtering_example.js';
+
+# Foundation Forms - Conditional Filtering
+
+Conditional filtering is a powerful feature that enables you to create dynamic, dependent form fields where the options of one field are filtered based on the selection of another field. This is particularly useful for implementing nested or cascading dropdowns, where each selection narrows down the available options in subsequent fields. Common use cases include country-city selection, counterparty-trader relationships, category-subcategory hierarchies, and department-employee mappings. This creates a more intuitive and guided user experience by preventing invalid selections and reducing the number of choices presented to the user.
```

Comment:
Nitpick but maybe put the common use cases in bullet points as that matches the style we've used in other docs.